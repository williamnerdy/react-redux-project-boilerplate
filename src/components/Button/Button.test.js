import React from 'react';
import { shallow } from 'enzyme';
import Button from './Button';

describe('<Button />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Button>My Button</Button>);
  });

  it('Renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
