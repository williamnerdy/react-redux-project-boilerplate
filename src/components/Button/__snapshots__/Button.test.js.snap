// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> Renders without crashing 1`] = `
<button>
  My Button
</button>
`;
